spec/ysh-case.test.sh

```diff
@@ -45,6 +49,9 @@ three
 ## END
 
 #### case syntax, complex expressions
+# TODO: make this execute
```

Comment:
Please don't disable failures this way

I did that in the last commit in `test/spec.sh ysh-case` -- it has --osh-failures-allowed 3

That is better so we still see the red/green as a TODO